.env.example

```diff
@@ -1,11 +0,0 @@
-  
-REACT_ENVIRONMENT=
-REACT_APP_NAME="Alt Markets Limited"
-REACT_APP_WEBSITE=https://alt.ng
-REACT_APP_BASE_URL=
-REACT_APP_PAYSTACK_BASE_URL=
-REACT_APP_FLUTTER_PUBK=
-REACT_APP_FLUTTER_SECK=
-REACT_APP_FLUTTERWAVE_ENCRYPTION_KEY=
-REACT_APP_PAYSTACK_PUBK=
-REACT_APP_PAYSTACK_SECK=
```

Comment:
Don't delete the .env.example file. Ideally, you will leave this file incase any other person want to continue with the project. It is better to create a new .env file, put in your env variables and leave the .env.example